sys-apps/baselayout/baselayout-2.2-r1.ebuild

```diff
@@ -181,7 +181,7 @@ src_prepare() {
 	echo "LDPATH='${ldpaths#:}'" >> etc/env.d/00basic
 
 	# rc-scripts version for testing of features that *should* be present
-	echo "Gentoo Base System release ${PV}" > etc/gentoo-release
+	echo "${DISTRO} Base System release ${PV}" > etc/gentoo-release
```

Comment:
Not sure about this either. I'd say that if you fork Gentoo, you want to fork baselayout to have proper *-release files for your distro. Maybe we should just assert for Gentoo here.